requirements.in

```diff
@@ -10,4 +10,5 @@ gunicorn
 dj-database-url
 psycopg2
 whitenoise
-sentry-sdk
\ No newline at end of file
+sentry-sdk
+ftd-django
```

Comment:
Let's use exact version number here.

Reply:
We will be changing this version number often so we need it explicitly spelled out.